### PR TITLE
add four blogs and update two old blogs tags

### PR DESCRIPTION
**Pull request by KID-G**
Changes:

add four blogs and update two old blogs tags

### CODE REVIEW

File: blog/zh/blog/2022/12/16/amazonlambda-with-apisix.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+`aws-lambda` 插件用于将 AWS Lambda 作为动态上游集成至 APISIX，从而实现将访问指定 URI 的请求代理到 AWS 云。用户使用该插件终止对已配置 URI 的请求，并代表客户端向 AWS Lambda Gateway URI 发起一个新的请求。
+
+这个新请求中携带了之前配置的授权详细信息，包括请求头、请求体和参数（以上参数都是从原始请求中传递的），之后 `aws-lambda` 插件会将带有响应头、状态码和响应体的响应信息返回给使用 APISIX 发起请求的客户端。该插件支持通过 AWS API key 和 AWS IAM secrets 进行授权。 插件细节可参考[官方文档](https://apisix.apache.org/zh/docs/apisix/plugins/aws-lambda)或者[博客](https://blog.bisakh.com/blog/aws-lambda-apisix)。
+
+### **Serverless 相关插件汇总**
+
+除了 Amazon Lambda，Apache APISIX 目前还支持与 Azure Function、Lua 函数和 Apache OpenWhisk 等 Serverless 相关生态的集成，从而提供了相应的 Serverless 插件，具体如下表所示。
+
+表格 还在加载中，请等待加载完成后再尝试复制
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/12/16/amazonlambda-with-apisix.md]: this part should be form or form image

File: blog/zh/blog/2022/12/19/auth-apisix-gateway.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+### **ACL 访问控制**
+
+在一个复杂的后端系统中，可能会存在部分 API 的安全限制是高于其他 API 的，这种限制不仅需要拦截匿名用户，而且需要对认证用户进行限制，例如：只允许白名单用户访问用户管理 API。
+
+![3322406862.jpg](https://static.apiseven.com/2022/12/22/63a4066e6b7e4.jpg)
+
+此时我们可以使用 APISIX 提供的 `consumer-restriction` 插件去实现一个访问控制机制。
+
+```plain
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/12/19/auth-apisix-gateway.md]:
```suggestion
```shell
```

File: blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
+---
+title: "为什么 APISIX Ingress 是比 Traefik 更好的选择？"
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md]: this file name is same with lamda, need update

File: blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+作为网关，最为核心的能力便是要能够正确的代理流量。作为 Kubernetes 集群的入口网关，主要处理如下两部分的流量：即 **Client 到网关的流量**和**网关与 Upstream 的流量**。如下所示：
+
+```bash
+Client <----> Ingress <----> Upstream Service
+```
+
+当前的协议多种多样，以下简单汇总了两个项目对协议的支持，仅供参考。
+
+（如 md 文件可引用下方格式）
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md]: delete this

File: blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+| zookeeper |           支持           |     支持     |
+|    DNS    |           支持           |    不支持    |
+
+此外，这两个项目都非常积极与一些周边项目进行了集成与合作。比如 Rancher、KubeSphere 等。
+
+从生态合作角度来看，APISIX Ingress 比 Traefik 提供了更为广泛的集成能力，尤其是与基础组件。因此在进行技术选型时，可以结合当前自己所用的基础组件的情况进行权衡。
+
+## 来自用户的声音
+
+在今年，我们也看到了很多来自用户的声音，他们开始在业务架构中用上了 APISIX Ingress。比如 [地平线使用 APISIX Ingress 替换了 Traefik](https://www.apiseven.com/usercase/apisix-ingress-with-horizon-ai)，主要是考虑如下方面：
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md]:
```suggestion
在今年，我们也看到了很多来自用户的声音，他们开始在业务架构中用上了 APISIX Ingress。比如[地平线使用 APISIX Ingress 替换了 Traefik](https://www.apiseven.com/usercase/apisix-ingress-with-horizon-ai)，主要是考虑如下方面：
```

File: blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+* 通过 Annotation 增加的配置不易重用；
+* Traefik 中默认的行为与 NGINX 中不同，用户在使用时候会产生困惑；
+
+在切换为 Apache APISIX Ingress 后，得益于 APISIX Ingress 丰富的插件生态，绝大多数需求均可通过内置插件满足。并且插件的配置可直接通过 APISIX Ingress 的 ApisixRoute 资源进行定义，比较直观。也可以通过 ApisixPluginConfig 进行插件模板的配置，在其他的 ApisixRoute 资源中进行引用。
+
+![image (8).png](https://static.apiseven.com/2022/12/22/63a4011134069.png)
+
+APISIX Ingress 的数据面性能更佳，能高效地应对日益增长的业务流量，而不会陷入性能瓶颈。
+
+除地平线以外，包括 [少年得到](http://www.igetcool.com/)，[观为智慧](http://www.gwwisdom.com/) 等公司也都使用 APISIX Ingress 替换了 Traefik，更多用户案例请参考 [用户案例](https://www.apiseven.com/usercases)。
```
Comment (SylviaBABY) [inline blog/zh/blog/2022/12/19/amazonlambda-with-apisix copy.md]:
```suggestion
除地平线以外，包括[少年得到](http://www.igetcool.com/)，[观为智慧](http://www.gwwisdom.com/)等公司也都使用 APISIX Ingress 替换了 Traefik，更多用户案例请参考[用户案例](https://www.apiseven.com/usercases)。
```

**Review by SylviaBABY — APPROVED** [review]
LGTM

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**